### ISSUE TITLE
[Bug] accessibility problems with talkback

### ISSUE THREAD

**Issue by mitch-hannigan** [spocky/miproja1#325]
**Describe the bug**
When using TalkBack as a screen reader, it cannot read app cards and other elements on the main screen.
It reads the top bar just fine — I mean elements like settings, connectivity, notifications, and the clock.
It also reads the names of categories, like "Mobile Apps," for example, but it can't read the app names themselves.
All other screens seem to be accessible.

**To Reproduce**
1. Enable the TalkBack screen reader.
2. Go to the home screen using the launcher.
3. Navigate around — TalkBack will read the elements as "No label."

**Expected behavior**
If the app sets accessibility properties correctly, TalkBack should read elements by their name or accessibility label.
It seems that the accessibility properties are not being set by the app for most elements on the main screen.

**Screenshots**
N/a

**Device:**
 - Device: raspberry pi 4b
 - Android version: android tv 15 (LineageOS)
 - Projectivy version 4.65

**How is Projectivy set as your default Launcher**
- Stock launcher disabled yes
- Projectivy 's accessibility service enabled
- Launcher Manager

**Additional context**

**Comment by spocky** [comment]
Fixed in 4.70
